debian/rules

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/make -f
+# See debhelper(7) (uncomment to enable)
+# output every command that modifies files on the build system.
+#export DH_VERBOSE = 1
+
+clean:
+	@# Do nothing
+
+build:
+	@# Do nothing becuase all source codes are written by a script language.
+
+binary:
+	mkdir -p    debian/taos-ci
+	mkdir -p    debian/taos-ci/var/www/html/TAOS-CI
+	cp -arf ./ci debian/taos-ci/var/www/html/TAOS-CI/
+	dh_gencontrol
+	dh_builddeb
```

Comment:
uh...

Doesn't
```
%:
        dh $@
```
fit for your needs here? (You usually don't need clean, build section specified)

It'd be better to have a common install script for both Tizen and Ubuntu (e.g., CMaktLists.txt with INSTALL sections). Having too specific script in debian/rules isn't too good for porting.

Reply:
We do not need a build section becaue TAOS-CI is written by Script languages (e.g., PHP, Bash, and Python). So, this PR is not used CMakeList.txt by default.